### PR TITLE
Commit prometheus-operator CRDs

### PR DESCRIPTION
**Pull request by vulkoingim**
## Description

Commits the `prometheus-operator` CRDs to the repo, so we don't have to generate them from jsonnet.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: installer/pkg/config/config.go [hunk 1]
```diff
@@ -29,10 +29,6 @@ func Defaults(in interface{}) error {
 		Install: false,
 	}
 
-	cfg.Prometheus = &Prometheus{
```
Comment (vulkoingim) [inline installer/pkg/config/config.go:32]: This is dup

**Review by ArthurSens — DISMISSED** [review]
If I understood it correctly, did you manually move CRDs to a new folder?

Would it make sense to make jsonnet generate CRDs directly in this new folder so we can keep them up-to-date with upstream?

**Comment by ArthurSens** [comment]
I can continue this PR if the comment above makes sense 🙂

**Comment by vulkoingim** [comment]
Yeah I moved them to a new folder so diffs are easier and also jsonnet generates the operator deployment as well - and we already have that in the obs installer.

**Comment by ArthurSens** [comment]
> Yeah I moved them to a new folder so diffs are easier and also jsonnet generates the operator deployment as well - and we already have that in the obs installer.

We can make jsonnet generate only the CRDs into this new folder and leave the prometheus-operator manifests right where they are

**Comment by ArthurSens** [comment]
@vulkoingim what do you think about the change I've made? 🙂

**Pull request merged**